src/status_im/chat/views/message/message.cljs

```diff
@@ -161,10 +124,10 @@
 (defn get-style [string]
   (->> replacements
        (into [] (comp
-                  (map first)
-                  (map #(vector % (re-pattern %)))
-                  (drop-while (fn [[_ regx]] (not (re-matches regx string))))
-                  (take 1)))
+                 (map first)
```

Comment:
alignement was right before, but just put the first argument on same line as comp and it will align fine